scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -266,6 +275,10 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
   def scalaJSOptimizer: Target[Boolean] = T { true }
 
+  def scalaJSImportMap: Target[Seq[ESModuleImportMapping]] = T {
```

Comment:
@lefou @lihaoyi
Suggestions for this name?

Reply:
What's wrong with the current name? What's the name of the underlying concept?